### ISSUE TITLE
Deprecated Warning On NPM Install - Node v4.2.6

### ISSUE THREAD

**Issue by langtutheky** [audreyt/node-webworker-threads#94]
I received this warning when npm install today. I am using Node v4.2.6. Should I be concerned?

```
CXX(target) Release/obj.target/WebWorkerThreads/src/WebWorkerThreads.o
In file included from ../src/WebWorkerThreads.cc:17:
../src/bson.cc:796:24: warning: 'DecodeBytes' is deprecated: Use DecodeBytes(isolate, ...) [-Wdeprecated-declarations]
        intptr_t len = DecodeBytes(info[0], BINARY);
                       ^
~/.node-gyp/4.2.6/include/node/node.h:318:32: note: 'DecodeBytes' has been explicitly marked deprecated here
                inline ssize_t DecodeBytes(
                               ^
~/.node-gyp/4.2.6/include/node/node.h:66:42: note: expanded from macro 'NODE_DEPRECATED'
    __attribute__((deprecated(message))) declarator
                                         ^
In file included from ../src/WebWorkerThreads.cc:17:
../src/bson.cc:803:9: warning: 'DecodeWrite' is deprecated: Use DecodeWrite(isolate, ...) [-Wdeprecated-declarations]
        DecodeWrite(data, len, info[0], BINARY);
        ^
~/.node-gyp/4.2.6/include/node/node.h:331:32: note: 'DecodeWrite' has been explicitly marked deprecated here
                inline ssize_t DecodeWrite(char* buf,
                               ^
~/.node-gyp/4.2.6/include/node/node.h:66:42: note: expanded from macro 'NODE_DEPRECATED'
    __attribute__((deprecated(message))) declarator
                                         ^
In file included from ../src/WebWorkerThreads.cc:17:
../src/bson.cc:897:28: warning: 'Encode' is deprecated: Use Encode(isolate, ...) [-Wdeprecated-declarations]
                Local<Value> bin_value = Encode(serialized_object, object_size, BINARY)->ToString();
                                         ^
~/.node-gyp/4.2.6/include/node/node.h:299:45: note: 'Encode' has been explicitly marked deprecated here
                inline v8::Local<v8::Value> Encode(
                                            ^
~/.node-gyp/4.2.6/include/node/node.h:66:42: note: expanded from macro 'NODE_DEPRECATED'
    __attribute__((deprecated(message))) declarator
                                         ^
3 warnings generated.
  SOLINK_MODULE(target) Release/WebWorkerThreads.node
webworker-threads@0.6.2 node_modules/webworker-threads
├── bindings@1.2.1
└── nan@2.2.0
```

**Comment by audreyt** [comment]
Not at the moment, but it would be good to update `WebWorkerThreads.cc` to conform to latest `nan.h`

**Comment by davisjam** [comment]
This warning was fixed in commit 4d1634fc. This issue should be closed.
